src/scripts/validation.js

```diff
@@ -17,6 +17,16 @@ const setEventListeners = (formEl, settings) => {
     inputEl.addEventListener("input", function () {
       checkInputValidity(formEl, inputEl, settings);
       toggleButtonState(inputList, buttonEl, settings);
+
+      const defaultText = formEl.querySelector(`#${inputEl.id}-default-text`);
+
+      if (inputEl.value.length > 2) {
+        defaultText.classList.remove(settings.errorClass);
+      } else if (inputEl.value.trim() !== "") {
+        defaultText.classList.add(settings.errorClass);
+      } else {
+        defaultText.classList.remove(settings.errorClass);
```

Comment:
* When I type anything in the inputs, there are errors https://snipboard.io/DK8Azn.jpg 


Please, delete this new code about `defaultText `. It's not needed inside `validtion.js`. Everything works fine without it